ironmq/src/main/scala/akka/stream/alpakka/ironmq/javadsl/CommittableMessage.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.ironmq.javadsl
+
+import java.util.concurrent.CompletionStage
+
+import akka.Done
+import akka.stream.alpakka.ironmq.Message
+
+/**
+ * Commit an offset that is included in a [[CommittableMessage]].
+ */
+trait Committable {
+  def commit(): CompletionStage[Done]
+}
+
+/**
+ * A [[Committable]] wrapper around the IronMq [[Message]].
+ */
+trait CommittableMessage extends Committable {
+  def message: Message
+}
```

Comment:
It would be great to have a simple test written in Java that would validate if the Java Dsl is alright.